src/Ocelot/Configuration/Creator/ITimeoutCreator.cs

```diff
@@ -0,0 +1,9 @@
+using Ocelot.Configuration.File;
+
+namespace Ocelot.Configuration.Creator
+{
+    public interface ITimeoutCreator
+    {
+        int Create(FileRoute fileRoute, FileGlobalConfiguration globalConfiguration);
```

Comment:
@ggnaegi @hogwartsdeveloper 
Why not to consider to move this tiny business logic method to [IRoutesCreator](https://github.com/search?q=repo%3AThreeMammals%2FOcelot%20IRoutesCreator&type=code) interface like this:
```C#
int CreateTimeout(FileRoute fileRoute, FileGlobalConfiguration globalConfiguration);
```